src/explore-education-statistics-admin/src/pages/release/edit-release/data/ReleaseFootnotesSection.tsx

```diff
@@ -82,22 +82,34 @@ const ReleaseFootnotesSection = ({
     <>
       <h2>Footnotes</h2>
 
+      <p>
```

Comment:
We could potentially end up in a state where `p` is empty, i.e. when conditions are true. This isn't great in terms of making the markup semantic.

Wrap each message with `p` to avoid this happening.

Reply:
Yep - done - Thanks